examples/ssh/opkssh/cli.go

```diff
@@ -165,6 +165,9 @@ func authorizedKeysCommand(userArg string, typArg string, certB64Arg string, pol
 
 func createSSHCert(cxt context.Context, client *client.OpkClient, signer crypto.Signer, alg jwa.KeyAlgorithm, gqFlag bool, principals []string) ([]byte, []byte, error) {
 	pkt, err := client.Auth(cxt, signer, alg, map[string]any{}, gqFlag)
+	if err != nil {
+		return nil, nil, err
+	}	
```

Comment:
```suggestion
	}
```
I think an extra space after the `}` is making the linter test fail, This should fix it.